src/components/ColumnsEditor.tsx

```diff
@@ -54,7 +55,7 @@ const makeFullInstructions = (raw) => [{symbol: 'begin'}, ...raw];
 const EmptyInstructions = makeFullInstructions([]);
 const transformInstructions = (raw) => JSON.stringify(raw);
 
-export const serverGetTransformRequester = (setDf) => {
+export const serverGetTransformRequester = (setDf) => {//
```

Comment:
This is mostly useful for development against a running py server as opposed to running insde the jupyter notebook with a widget.